src/lib.rs

```diff
@@ -144,20 +146,18 @@ where
     }
 
     // Decrypt the private key bytes using AES-128-CTR
-    let mut decryptor = Aes128Ctr::new(
-        (&key[..16]).into(),
-        (&keystore.crypto.cipherparams.iv[..16]).into(),
-    );
+    let decryptor =
+        Aes128Ctr::new(&key[..16], &keystore.crypto.cipherparams.iv[..16]).expect("invalid length");
 
-    let mut pk = keystore.crypto.ciphertext.clone();
+    let mut pk = keystore.crypto.ciphertext;
     decryptor.apply_keystream(&mut pk);
 
     Ok(pk)
 }
 
 /// Encrypts the given private key using the [Scrypt](https://tools.ietf.org/html/rfc7914.html)
 /// password-based key derivation function, and stores it in the provided directory. On success, it
-/// returns the `id` (Uuid) generated for this keystore.
+/// retur ns the `id` (Uuid) generated for this keystore.
```

Comment:
I think this is a typo?

Reply:
Oops, not sure how that happened. Fixed.